src/mscorlib/src/Interop/Windows/kernel32/Interop.Globalization.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
There is a number of no-diff changes like this one. I assume that it is a step in the right direction...